src/spec.md

```diff
@@ -170,6 +170,10 @@ try {
 }
 ```
 
+Note that you need to trigger this api from a user interaction,
+for example when clicking on a button,
+because some implementations, such as the basic simulator (`webxdc.js`) use the `<input type=file>` element under the hood.
```

Comment:
so this is how we ended with a "broken" API up-front, sweet :)

Reply:
you know that the input element has the same problem? you can only do clicks via js when that js code was triggered by a user action

Reply:
the "because" does not really explain the reason:

```suggestion
because some implementations use`<input>` elements under the hood that require user interaction as well.
```